.jenkins/infrastructure/docker/dockerfiles/linux/Dockerfile

```diff
@@ -64,6 +64,34 @@ ARG devkits_uri
 # Check to make sure devkits_uri is defined as build arg
 RUN test ! -z ${devkits_uri+x}
 
+# Install essential packages
+RUN apt-get update
+RUN apt-get upgrade -y
+RUN apt-get -y install make build-essential git jq vim curl wget netcat
+
+# Add Microsoft repo
+RUN echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ bionic main" | tee /etc/apt/sources.list.d/azure-cli.list
+RUN wget https://packages.microsoft.com/keys/microsoft.asc
+RUN apt-key add microsoft.asc
+RUN apt-get update
+
+# Install Microsoft packages
+RUN apt-get -y install apt-transport-https azure-cli unzip
+
+# Install terraform
+RUN wget https://releases.hashicorp.com/terraform/0.12.18/terraform_0.12.18_linux_amd64.zip
+RUN unzip terraform_0.12.18_linux_amd64.zip -d /usr/sbin
+RUN rm terraform_0.12.18_linux_amd64.zip
+
+# Install oe-engine
+RUN curl https://oejenkinsciartifacts.blob.core.windows.net/oe-engine/latest/bin/oe-engine -o /usr/bin/oe-engine
```

Comment:
We no longer use oe-engine either, as Azure Confidential COmputing has gone into public preview with their DC skus and we are using those.